R/BASiCS_DenoisedCounts.R

```diff
@@ -52,7 +52,7 @@ BASiCS_DenoisedCounts <- function(Data, Chain) {
       CountsBio <- counts(Data)
       CountsTech <- assay(altExp(Data))
       Phi <- matrixStats::colMedians(Chain@parameters$phi)
-      out1 <- t(t(CountsBio) / (Phi * Nu))
+      out1 <- t(t(CountsBio) / matrixStats::colMedians(Chain@parameters$phi * Chain@parameters$nu))
```

Comment:
This is not the same as it was before and therefore I'd expect that the unit test would fail. 

Before we first calculated posterior medians for `Phi` and `Nu` separately. Instead, you calculate the posterior median for `Phi * Nu`. These two quantities are not equivalent and therefore I suggest to revert to what it was.

Reply:
I got this idea from an expert in the field, to be fair :) https://github.com/catavallejos/BASiCS/issues/182

Reply:
🤣 🤣 🤣 

on a second thought ... who am I to disagree with such an eminent person!

Reply:
I think the change is ok. The relevant parameter for that calculation is the product between phi and nu!